process-engine/bpmn-studio
# ✨Show Notification if Open Diagram Changed on Filesystem

## Changes

1. Add notification that gets displayed when an open diagram changed on filesystem

## Issues

Closes #1605

PR: #1618

## How to test the changes

- Open some diagrams
- Change/move the files without using the BPMN-Studio
- **Notice that notifications will be displayed as soon as changes happen.**

- Change/rename/delete the diagrams via the BPMN-Studio
- **Notice that no notification will be displayed.**

- Open a diagram
- Move the diagram
- Make some changes without using the BPMN-Studio
- **Notice that no notification will be displayed.**
- Move the diagram back
- Make some changes without using the BPMN-Studio
- **Notice that a notification will be displayed.**

## Known Bugs

- (Vorherige) Xml wird nicht aktualisiert wenn lokal Änderungen auftreten (#1627)
	- Diff View zeigt Änderungen nicht richtig an
	- Edited Symbol (SolutionExplorer) verschwindet unter Umständen nach Diagram wechsel
		- Open Diagram auswählen (ohne Edited Flag)
		- Änderungen außerhalb des BPMN-Studios machen
		- Edited Symbol (SolutionExplorer) erscheint
		- Diagram wechseln
		- Edited Symbol (SolutionExplorer) verschwindet
